salt/modules/vsphere.py

```diff
@@ -2077,17 +2080,19 @@ def list_clusters(host, username, password, protocol=None, port=None):
         salt '*' vsphere.list_clusters 1.2.3.4 root bad-password
 
     '''
-    service_instance = salt.utils.vmware.get_service_instance(host=host,
-                                                              username=username,
-                                                              password=password,
-                                                              protocol=protocol,
-                                                              port=port)
+    if any([host, username, password, protocol, port]):
+        service_instance = salt.utils.vmware.get_service_instance(host=host,
+                                                                  username=username,
+                                                                  password=password,
+                                                                  protocol=protocol,
+                                                                  port=port)
```

Comment:
Same here. I think it's entirely appropriate to just make it a unit test and mock out `salt.utils.vmware.list_clusters`.